packages/itwinui-css/backstop/tests/surface.html

```diff
@@ -52,6 +55,35 @@
       @media (max-width: 600px) {
         #demo-elevations { flex-direction: column; }
       }
+      .demo-surface-list {
+        all: unset;
+        display: flex;
+        flex-direction: column;
+        flex: 1;
+      }
+      .demo-surface-list li {
+        padding-block: var(--iui-size-s);
+        padding-inline: var(--iui-surface-padding-inline);
+        position: relative;
+      }
+      .demo-surface-list li:not(:first-of-type) {
+        border-top: 1px solid var(--iui-surface-border-color);
+      }
+      .demo-surface-list li a::after {
+        content: '';
+        position: absolute;
+        inset: 0;
+      }
+      .demo-mayanks-flex-component {
+        display: flex;
+        flex-grow: 1;
+        justify-content: space-between;
+      }
+      .demo-mayanks-flex-component-2 {
```

Comment:
@mayank99 I'd like your input on line 82 & 77 being used in here.  I'm suggesting that we do not build this styling into the component and instead expect the user to piece together with your flex component.

Reply:
yup, this component should be as minimal as possible.